libs/wire-api/src/Wire/API/Routes/Public.hs

```diff
@@ -61,6 +61,8 @@ type ZConn = ZAuthServant 'ZAuthConn InternalAuthDefOpts
 
 type ZOptUser = ZAuthServant 'ZAuthUser '[Servant.Strict]
 
+type ZOptConn = ZAuthServant 'ZAuthConn '[Servant.Optional, Servant.Strict]
```

Comment:
Could we either add an explicit `Optional` to the type above as well, for consistency? Or alternatively remove it from this one.

Reply:
I prefer it explicit; done in dc6ea077565ab523dad5d9713bfb5d9ef163734d.